.gitignore

```diff
@@ -17,3 +17,8 @@
 /.tshy
 /.tshy-*
 /__tests__
+/examples/node_modules
+/coverage
+/bun.*
+/examples/bun.*
+/examples/deno.lock
```

Comment:
if you're keeping the `package-lock.json`, you might as well keep the deno one ;)